content/blog/crossing-realities-tips-and-tricks.md

```diff
@@ -0,0 +1,107 @@
+
+But my experience of hacking the Daydream *Input Module* was telling me that building that kind of thing would be really challenging. Hell, it might not even be possible at all for what I knew! But yet, if it does work then it will save use a lot of time and troubles, so let's try!
+
+And there I went, diving so deep in Unity that I reached parts where there is no documentation anymore (and for Unity it means you have gone wayyyyy out of the beaten tracks). After a few hours of digging and testing I started to get a clearer picture of the monster hiding behind Unity's abstraction layers. It was still unclear at that time if I could tame the beast, but I had gone to far to step back, so I went on. Going through Unity source code to better analyze my opponent I put together a strategy. Inheriting where I could and copy-pasting where I had to, refining at every new iteration, I slowly crafted my tools, preparing for the final assault. And there I went, surrounding the mighty creature with my interfaces, luring it with my classes. I got bitten a few times but slowly it resistance lowered, until we started to understand each other. Crafting our own words we managed to communicate, understand each other needs, and fulfil them. We made it, the magic happened.
+
+But enough with the tales of my adventures, here is what you can do with this *Input Module*.
+
+### The uses
+If you are familiar with Unity [*Event Triggers*](https://docs.unity3d.com/Manual/script-EventTrigger.html) you know that you can use them to interact with *GameObjects* using your mouse. Google have created an *Input Module* to detect gazed based event for the Cardboard, which is pretty handy to create cool interactions :). And if you don't know *Event Triggers* then you definitely have to [watch this tutorial](https://unity3d.com/learn/tutorials/topics/user-interface-ui/ui-events-and-event-triggers) by Unity!
+
+![article-img-centered](/img/blog/0014/event_trigger_mouse.PNG "Mouse Event Trigger")
+<center>Mouse Event Trigger</center>
+
+What Ghosthands does is giving you the same flexibility by offering you a range of events to listen to for the Oculus Touch controllers.
+
+![article-img-centered](/img/blog/0014/event_trigger_oculus_touch.PNG "Oculus Touch Event Trigger")
+<center>Ghosthands Event Trigger</center>
+
+What it **doesn't do** (yet) is send events to UI elements that don't have a collider (but if you add a collider it will work fine).
+
+### The setup
+You don't have much to do to use this plugin, it the same setup as using Unity *Event System*:
```

Comment:
it the same setup=> it IS the same setup
as using Unity *Event System*: =>as Unity *Event System*: